apps/sites/app/[lang]/[id]/page.tsx

```diff
@@ -8,21 +8,31 @@ import { useEffect, useState } from 'react';
 import { P2, Select, Tabs } from 'ui/core';
 import { TabItem } from 'ui/core/tabs/Tabs.types';
 import { twMerge } from 'tailwind-merge';
-import { SiteForm } from '@components/form/site/SiteForm';
+import { SiteForm } from '@app/[lang]/[id]/components/form/site/SiteForm';
```

Comment:
```suggestion
import { SiteForm } from './components/form/site/SiteForm';
```
Maybe I'm wrong, please check